src/pages/VenuePage/VenuePage.tsx

```diff
@@ -74,11 +73,15 @@ const VenuePage = () => {
     return <>This event does not exist</>;
   }
 
-  if (!event || (event.price > 0 && !eventPurchase) || !venue || !users) {
+  if (!event || !venue || !users || !userPurchaseHistoryRequestStatus) {
```

Comment:
Should we check here that the event is actually ongoing (based on the duration)?

Reply:
In theory we should, but I'd be nervous about any re-render kicking someone out of the bar in the middle of their deepest philosophical moment :)

Reply:
Recommendation is to not kick people out of the bar for now